package.json

```diff
@@ -1,45 +1,40 @@
 {
-  "name": "project_base",
+  "name": "sustainability_kiosk",
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "SET NODE_OPTIONS=--openssl-legacy-provider && vue-cli-service serve --open",
-    "build": "node --max_old_space_size=4096 node_modules/.bin/vue-cli-service build --mode=production",
-    "build-stage": "node --max_old_space_size=4096 node_modules/.bin/vue-cli-service build --mode staging",
-    "lint": "vue-cli-service lint",
-    "prettier": "prettier --write **/*.{js,ts,tsx,json,html,md,css,scss,vue}",
+    "dev": "vite",
+    "build": "vite build",
+    "build-stage": "vite build --mode staging",
+    "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix src",
+    "prettier": "prettier --write **/*.{js,json,html,md,css,scss,vue}",
     "format": "npm-run-all --continue-on-error prettier \"lint -- --fix\""
   },
   "dependencies": {
-    "axios": "^0.19.2",
+    "autoprefixer": "^10.4.20",
+    "axios": "^1.7.9",
     "cache-loader": "^4.1.0",
```

Comment:
This can be removed now (and should clear at least one of the remaining warnings), it was needed for Vue CLI but is no longer needed with Vite.